demo/backend/index-reloaded.xml.njk

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
to prevent complete duplication of the navigation markup from index.xml, could we:
- add template blocks in index.xml to allow setting the selected tab
- make this template extend index.xml and set the relevant block to select the desired tab

Reply:
https://github.com/Instawork/hyperview/pull/1005/commits/54c0f7fd4a3b41884e760f1bab82bcd40f90e62d